### DISCUSSION TITLE
Embedded Milvus running in browser with PyScript?

### DISCUSSION THREAD

**Discussion by NOT-HAL9000** [milvus-io/milvus#19013] [Ideas & Feature requests]
Just wondering if it would be possible to get a lightweight embedded version running in browser with the recently released PyScript? 

Thanks!

**Comment by yhmo** [comment]
We install an embedded milvus by these steps: https://milvus.io/docs/v2.1.x/install_embedded_milvus.md
When we call milvus.start(), it starts a standalone server in a process. The standalone server stores data in the folder /var/bin/e-milvus.

When you write PyScript code in HTML, it is interpreted by the browser which embeds a python interpreter. If the interpreter has the ability to start a process, then it is possible to run milvus in a browser. I didn't get a chance to try, you can have a try.
